src/Raven.Client/ServerWide/ExternalReplication.cs

```diff
@@ -10,8 +11,19 @@ public class ExternalReplication : ReplicationNode, IDatabaseTask, IDynamicJsonV
     {
         public long TaskId;
         public string Name;
+        public string[] TopologyDiscoveryUrls;
         public string MentorNode;
 
+        public ExternalReplication() { }
+
+        public ExternalReplication(string[] urls)
```

Comment:
where is the db that we'll be replicating to?

Reply:
the Database property of ReplicationNode class